src/java/Employee.java

```diff
@@ -0,0 +1,57 @@
+class Employee {
+    private String FullName;
+    private String Position;
+    private String Email;
+    private int TelNumber;
```

Comment:
Please rename `TelNumber` to `phoneNumber`. Rename getter and setter for this variable as well. and constructor parameter `telNumber` should be renamed too.